### PR TITLE
Provide a facility for registering encoder services

### PR DESCRIPTION
**Pull request by GrabYourPitchforks**
/cc @anurse

### CODE REVIEW

File: src/Microsoft.AspNet.Http.Extensions/Encoders/EncoderServices.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+using System;
+using System.Collections.Generic;
+using Microsoft.Framework.ConfigurationModel;
+using Microsoft.Framework.DependencyInjection;
+
+namespace Microsoft.AspNet.WebUtilities.Encoders
+{
+    public static class EncoderServices
+    {
+        public static IEnumerable<IServiceDescriptor> GetDefaultServices(IConfiguration configuration = null)
```
Comment (davidfowl) [inline src/Microsoft.AspNet.Http.Extensions/Encoders/EncoderServices.cs]: No more optional parameters.

Comment (GrabYourPitchforks) [inline src/Microsoft.AspNet.Http.Extensions/Encoders/EncoderServices.cs]: Would the parameterless overload just call the parameterized overload passing null?

Comment (davidfowl) [inline src/Microsoft.AspNet.Http.Extensions/Encoders/EncoderServices.cs]: Good question, yes pass null.

Comment (GrabYourPitchforks) [inline src/Microsoft.AspNet.Http.Extensions/Encoders/EncoderServices.cs]: Got it - thanks. :)

File: src/Microsoft.AspNet.Http.Extensions/Encoders/EncoderServiceCollectionExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.Http;
+using Microsoft.Framework.ConfigurationModel;
+using Microsoft.Framework.DependencyInjection;
+
+namespace Microsoft.AspNet.WebUtilities.Encoders
```
Comment (Praburaj) [inline src/Microsoft.AspNet.Http.Extensions/Encoders/EncoderServiceCollectionExtensions.cs]: in `Microsoft.Framework.DependencyInjection` namespace like rest other `IServiceCollection` extensions?

Comment (GrabYourPitchforks) [inline src/Microsoft.AspNet.Http.Extensions/Encoders/EncoderServiceCollectionExtensions.cs]: Got it. :)

**Comment by Praburaj** [comment]
@ajaybhargavb

**Pull request merged**